modules/file-system/parallelstore/variables.tf

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  description = "Project in which the HPC deployment will be created."
+  type        = string
+}
+
+variable "deployment_name" {
+  description = "Name of the HPC deployment."
+  type        = string
+}
+
+variable "instance_id" {
+  description = "Name of parallelstore instance."
+  type        = string
+  default     = "parallelstore-instance"
```

Comment:
why provide default? wouldn't it lead to collisions ?

Reply:
This was for POC local testing purpose. Updated.